core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -40,6 +42,8 @@ describe('Topics and skills dashboard functionality', function() {
       new SkillEditorPage.SkillEditorPage();
     topicEditorPage =
       new TopicEditorPage.TopicEditorPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
```

Comment:
`var explorationEditorMainTab`
Also, If we are only going to use the main page then can we inject `ExplorationEditorMainTab` (not sure)?

Reply:
I'm not sure of that. This was the format followed in other exploration tests as well. There is no separate `ExplorationEditorMainTab.js` file.